firebase/db_operate.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import os
+
+import firebase_admin
+from dotenv import load_dotenv
+from firebase_admin import credentials, db, firestore
+
+load_dotenv()
+
+cred = credentials.Certificate("firebase/cred.json")
+
+firebase_admin.initialize_app(
+    cred,
+    {
+        "databaseURL": os.getenv("FIREBASE_URL"),
+        "databaseAuthVariableOverride": {"uid": "my-service-worker"},
+    },
+)
+
+fireDB = firestore.client()
+
+
+def create_item(sente, gote) -> None:
+    # databaseに初期データを追加する
+    users_ref = db.reference()
+
+    realID = users_ref.push(["平手"])
```

Comment:
変数がキャメルケースになってるのでスネークケースに修正してください
あとID自体はrealID.keyに入ってる気がするので、命名調整して欲しいです〜。
```suggestion
    realtime_db_ref = users_ref.push(["平手"])
```